network/us-east-1/client-vpn/README.md

```diff
@@ -0,0 +1,132 @@
+# AWS Client VPN
+
+This layer deploy the AWS Client VPN service using AWS SSO to authenticate users.
+
+## Configure AWS SSO
+
+Due to API limitations, the SSO application must be configured manually (For more information regarding this limitation you can check [this bug](https://github.com/hashicorp/terraform-provider-aws/issues/34813) and the [notes in the AWS Terraform Provider](https://registry.terraform.io/providers/hashicorp/aws/5.39.1/docs/resources/ssoadmin_application?lang=typescript)).
+
+1. In the AWs Identity Center go to to “Applications”, then click the “Add a new application” button, then click “Add a custom SAML 2.0 application”.
+2. Complete the Name and description (if you will import this resource to terrraform note than you ewill not be able to change the name).
+3. Download the AWS SSO Metadata file (with name `saml-metadata.xml` in this folder, it will be ignored by git for of security)
```

Comment:
```suggestion
3. Download the AWS SSO Metadata file (with name `saml-metadata.xml` in this folder, it will be ignored by git for security reasons)
```